### PR TITLE
Limit caching to just our domains

### PR DESCRIPTION
**Pull request by amaliwanag2**

### CODE REVIEW

**Comment by amaliwanag2** [comment]
@kmjennison I had to enable "variables provided by system" in vercel in all environments to expose the VERCEL_URL env variable

**Comment by amaliwanag2** [comment]
@kmjennison there is an existing issue in offline mode.  If you navigate to account, then back to newtab it shows the loading screen as it trys to refetch data

**Comment by amaliwanag2** [comment]
also it looks like codecov/project is not ignoring the next config file when counting line coverage, not that it matters that much I just don't like the x lol

File: next.config.js [hunk 1]
```diff
@@ -10,7 +10,12 @@ const withSourceMaps = require('@zeit/next-source-maps')({
 })
 
 const basePath = process.env.NEXT_PUBLIC_URLS_BASE_PATH || ''
-
+const url = process.env.VERCEL_URL || 'http://localhost:3001/'
+const devAssetsRegex = 'https://prod-tab2017-media.gladly.io/.*'
+const prodAssetsRegex = 'https://dev-tab2017-media.gladly.io/.*'
+const cachingRegex = new RegExp(
```
Comment (kmjennison) [inline next.config.js:16]: We'll need this to work for our dev/prod domains, too. Let's just include the following, hardcoded:
`https://dev-tab2017.gladly.io/newtab/.*`
`https://tab.gladly.io/newtab/.*`

You can test by merging into the `dev` branch and making sure all works as expected [on dev](https://dev-tab2017.gladly.io/newtab/).

**Review by kmjennison — CHANGES_REQUESTED** [review]
Nice job, I like your straightforward approach here.

**Comment by kmjennison** [comment]
> @kmjennison there is an existing issue in offline mode. If you navigate to account, then back to newtab it shows the loading screen as it trys to refetch data

Got it. Definitely want to show an "offline" toast in the future if not making the app fully functional offline. That's in the backlog.

File: next.config.js (hunk 1 above)
Comment (amaliwanag2) [reply next.config.js:16]: @kmjennison I believe that the VERCEL_URL will resolve to those values at each of those environments

Comment (amaliwanag2) [reply next.config.js:16]: testing now on dev

**Pull request merged**